next/pages/vzn/[slug].tsx

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+
+import { ParsedUrlQuery } from 'node:querystring'
+
+import { GeneralQuery, RegulationEntityFragment } from '@backend/graphql'
+import { client } from '@backend/graphql/gql'
+import PageLayout from '@components/layouts/PageLayout'
+import RegulationDetailPageContent from '@components/pages/RegulationDetailPageContent'
+import { LocalizationsProvider } from '@components/providers/LocalizationsProvider'
+import PageHeader from '@components/ui/PageHeader/PageHeader'
+import { SectionContainer } from '@components/ui/SectionContainer/SectionContainer'
+import { GeneralContextProvider } from '@utils/generalContext'
+import { GetStaticPaths, GetStaticProps } from 'next'
+import Head from 'next/head'
+import * as React from 'react'
+
+interface StaticParams extends ParsedUrlQuery {
+  slug: string
+}
+
+type RegulationPageProps = {
+  general: GeneralQuery
+  regulation: RegulationEntityFragment
+}
+
+export const getStaticPaths: GetStaticPaths<StaticParams> = async () => {
+  const { regulations } = await client.allRegulations()
+
+  const paths = (regulations?.data ?? [])
+    .filter((regulation) => regulation?.attributes?.slug)
+    .map((regulation) => ({
+      params: {
+        slug: regulation?.attributes?.slug ?? '',
+      },
+    }))
+
+  console.log(`GENERATED STATIC PATHS FOR ${paths.length} SLUGS - REGULATIONS`)
+  return { paths, fallback: 'blocking' }
+}
+
+export const getStaticProps: GetStaticProps<RegulationPageProps, StaticParams> = async (ctx) => {
+  console.log(`Revalidating ${ctx}`)
+  const locale = ctx.locale ?? 'sk'
+  const slug = ctx.params?.slug ?? ''
+
+  const [{ regulations }, general, messages] = await Promise.all([
+    client.RegulationBySlug({ slug }),
+    client.General({ locale }),
+    import(`../../messages/${locale}.json`),
+  ])
+
+  if (!regulations || !regulations?.data?.length) {
+    return {
+      notFound: true,
+    } as const
+  }
+
```

Comment:
It would be nice to follow the pattern in other [slug] files, e.g.:
```tsx
export const getStaticProps: GetStaticProps<PageProps, StaticParams> = async ({
  locale,
  params,
}) => {
  const slug = params?.slug

  // eslint-disable-next-line no-console
  console.log(`Revalidating inba article ${locale === 'en' ? '/en' : ''}/inba/text/${slug}`)

  if (!slug || !locale) return { notFound: true }

  const [{ inbaArticles }, general, messages] = await Promise.all([
    client.InbaArticleBySlug({ slug, locale }),
    client.General({ locale }),
    import(`../../../messages/${locale}.json`),
  ])

  const inbaArticle = inbaArticles?.data[0]
  if (!inbaArticle) return { notFound: true }
```